src-electron/db/query-zcl.js

```diff
@@ -498,23 +498,17 @@ async function selectAllStructItemsByStructName(
   packageIds,
   clusterName = null
 ) {
-  let clusterJoinQuery = ''
   let clusterWhereQuery = ''
   if (clusterName) {
```

Comment:
With these changes, for a global struct none of the items are being shown.

To be clear, for a global struct what will come through here is some name for the struct _and some name for the cluster where the struct is being used_.  And when the lookup fails for that struct in that cluster, a lookup for a global struct with that name should happen....

Reply:
Cleaned this up. Thanks